docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
       - AWS_SECRET_ACCESS_KEY=arbitrary
 
   pathfinder-api:
-    image: mojdigitalstudio/pathfinder-api:latest
+    image: quay.io/hmpps/pathfinder-api:latest
```

Comment:
I assume this has been migrated to quay? It is difficult to keep track

Reply:
I think the pathfinder-api (as opposed to the main pathfinder project) started its life in quay

Reply:
So this is a correction really, rather than an update